third_party/terraform/tests/resource_bigquery_data_transfer_config_test.go

```diff
@@ -54,17 +57,21 @@ func testAccBigqueryDataTransferConfig_scheduledQuery_basic(t *testing.T) {
 
 func testAccBigqueryDataTransferConfig_scheduledQuery_update(t *testing.T) {
 	random_suffix := randString(t, 10)
+	first_start_time := time.Now().Add(1 * time.Hour).Format(time.RFC3339)
```

Comment:
These seem to in an incorrect format as the tests detect diffs based on them.

For example:
```
                  ~ schedule_options {
                        disable_auto_scheduling = false
                      ~ end_time                = "2020-11-12T17:39:51Z" -> "2020-11-12T09:39:51-08:00"
                      ~ start_time              = "2020-10-12T17:39:51Z" -> "2020-10-12T10:39:51-07:00"
                    }
```